packages/kit/src/runtime/client/router.js

```diff
@@ -109,12 +109,17 @@ export class Router {
 		const handle_mousemove = (event) => {
 			clearTimeout(mousemove_timeout);
 			mousemove_timeout = setTimeout(() => {
-				trigger_prefetch(event);
+				if (!event.target) return;
+				// ensure event.composedPath() in find_anchor(event) returns an array of EventTarget objects
+				event.target.dispatchEvent(
+					new CustomEvent('sveltekit:trigger_prefetch', { bubbles: true })
+				);
```

Comment:
```suggestion
				// ensure event.composedPath() in find_anchor(event) returns an array of EventTarget objects
				event.target?.dispatchEvent(
					new CustomEvent('sveltekit:trigger_prefetch', { bubbles: true })
				);
```

Using `?.` would be a bit cleaner.